lib/developer_portal/app/controllers/developer_portal/admin/account/passwords_controller.rb

```diff
@@ -16,7 +22,9 @@ def create
     end
   end
 
-  def new; end
+  def new
+    assign_drops(account: Liquid::Drops::Account.new(buyer))
```

Comment:
Do you need to expose this to the liquid?

You can just use the registers (current_account or site_account) if only used in ruby files and not liquid

https://github.com/3scale/porta/blob/3968319a9fa477de9997ad7860bbc8ab5268d465/lib/liquid/template_support.rb#L132-L133

Reply:
Does that mean I can use `current_account` directly without calling `assign_drop`?

Reply:
yes just use context.registers[:current_account] in the ruby code